Arrangement_on_surface_2/doc/Arrangement_on_surface_2/CGAL/Arr_conic_traits_2.h

```diff
@@ -440,13 +440,14 @@ class Arr_conic_traits_2 {
     Bbox_2 operator()(const X_monotone_curve_2& xcv) const { return bbox(xcv); }
   };
 
-  /*! \name Auxiliary Functor definitions, used gor, e.g., the landmarks
+  /*! \name Auxiliary Functor definitions, used gor, e.g., the landmarks \
```

Comment:
is the backslash here on purpose?

Reply:
Yes it is.

The code reads 
```
  /*! \name Auxiliary Functor definitions, used gor, e.g., the landmarks
   * point-location strategy and the drawing function.
   */
```
and this means that without the `\` the `\name` command terminates after the word "landmarks".

(I now see that in Arrangement_on_surface_2/include/CGAL/Arr_conic_traits_2.h the same construct exists, so for consistency I'll also update this file).